pipelines/batch/Dockerfile

```diff
@@ -34,15 +44,13 @@ ENV JDBC_FETCH_SIZE=10000
 ENV JDBC_MAX_POOL_SIZE=50
 ENV JDBC_INITIAL_POOL_SIZE=10
 ENV NUM_FILE_SHARDS=3
-ENV FHIR_DEBEZIUM_CONFIG_PATH="../utils/dbz_event_to_fhir_config.json"
+ENV FHIR_DEBEZIUM_CONFIG_PATH="../../utils/dbz_event_to_fhir_config.json"
 
-RUN echo 'Batch Job started'
-VOLUME [ "${WORK_DIR}" ]
 
-ENTRYPOINT java -cp app.jar org.openmrs.analytics.FhirEtl \
+ENTRYPOINT java -jar /usr/src/Main/app.jar  \
            --openmrsServerUrl=${OPENMRS_URL} --openmrsUserName=${OPENMRS_USERNAME} --openmrsPassword=${OPENMRS_PASSWORD} \
            --resourceList=${RESOURCE_LIST} --batchSize=${BATCH_SIZE} --targetParallelism=${TARGET_PARALLELISM} \
            --fhirSinkPath=${SINK_PATH}  --sinkUserName=${SINK_USERNAME} --sinkPassword=${SINK_PASSWORD} \
            --outputParquetPath=${PARQUET_PATH} --jdbcModeEnabled=${JDBC_MODE_ENABLED} --jdbcDriverClass=${JDBC_DRIVER_CLASS} \
            --jdbcMaxPoolSize=${JDBC_MAX_POOL_SIZE} --fhirDebeziumConfigPath=${FHIR_DEBEZIUM_CONFIG_PATH} \
-           --jdbcInitialPoolSize=${JDBC_INITIAL_POOL_SIZE} --jdbcFetchSize=${JDBC_FETCH_SIZE}
+           --jdbcInitialPoolSize=${JDBC_INITIAL_POOL_SIZE} --jdbcFetchSize=${JDBC_FETCH_SIZE}
```

Comment:
nit: It seems the only change in this line is the `\n` at the end, correct? Can you check your editor settings to see if we can avoid these kind of changes?

Reply:
Any thoughts about this?

Reply:
I think I deleted the blank line at the end. Re-adding it to see if it removes the diff